src/neo-vm/Types/Buffer.cs

```diff
@@ -40,22 +40,6 @@ public override StackItem ConvertTo(StackItemType type)
             }
         }
 
-        public override bool Equals(object obj)
-        {
-            return this == obj;
-        }
-
-        public override int GetHashCode()
-        {
-            unchecked
-            {
-                int hash = 17;
-                foreach (byte element in InnerBuffer)
-                    hash = hash * 31 + element;
-                return hash;
-            }
-        }
-
         public override bool ToBoolean()
         {
             return true;
```

Comment:
```suggestion
              if (Size > Integer.MaxSize) return true;
            return Unsafe.NotZero(InnerBuffer);
```
same logic as applied for `ByteString`.

Reply:
We are temporarily unable to reach a consensus on this issue, and the function here has not been modified, it is still the same as the original. So I will merge this PR first. If there is a problem, you can create a new issue separately. @ixje